mungegithub/mungers/lgtm_after_commit.go

```diff
@@ -66,7 +66,7 @@ func (LGTMAfterCommitMunger) Munge(obj *github.MungeObject) {
 		return
 	}
 
-	lastModified := obj.LastModifiedTime()
+	lastModified, _ := obj.LastModifiedTime()
```

Comment:
Can you split the error checking https://github.com/kubernetes/contrib/pull/1357/files#diff-d54eba901e3ff2426a7c69c8f323f129R72 to include `if err != nil || lastModified == nil`